docs/content/guidelines/usage.mdx

```diff
@@ -5,74 +5,93 @@ title: Usage guidelines
 
-### Octicons custom to `git`
+## Icon styling
 
-The following should only be used for their specific meanings:
+By default, all octicons use an outline style. Some icons have a filled version which should be used only under the following conditions:
 
-- Commits: The commit icon is represented as a circle on a line. This is to symbolize a point (the circle) along the version history timeline in which changes were made. This icon should be used in views to represent individual or groups of commits. These views include the timeline and the Commits tab in pull requests.
-- Issues: The issue icon is represented as a circle with an `!` mark. The base issue icon is extended through slight modifications to communicate different states of an issue: a checkmark for a closed issue and combining the sync icon with an `!` for reopening an issue. These three symbols should be used exclusively with issues.
-- Pull requests: The pull request icon is represented as a commit dot attached with an arrow being inserted into the history timeline (two dots connected by a line).
-- Repos: A repository in GitHub is represented as a closed book and is explicitly named as `repo`. Octicons does contain a `book` icon which displays an open book symbol that is used for wikis, READMEs, and other use cases where the user can see a collection of information.
+<Do>
+    <img width="456" alt="octicons-style-pairing" src="https://user-images.githubusercontent.com/10384315/103719162-490c9600-4f7d-11eb-8d9c-1d867ee51de9.png" />
+    <Caption>To aid in contrast when in relation to another icon</Caption>
+</Do>
 
-### Octicons custom to GitHub brand
-
-- Octoface: Represents the user. Conveys connection between GitHub and the user. Similar to the octocat generator.
-- GH mark: Brand logo use only. Follow brand guidelines.
-
-## Outline vs fill: What's the difference?
-
-By default, octicons have been designed in an outline style where filled shapes create small details (e.g. the repo icon where the book is outlined and the bookmark is filled). The use for using a fill style icon instead of an outline icon has been intentional and for the following use cases:
-
-- To aid in contrast when in relation to another icon [TODO: Screenshot of repo files view folder icon vs file icon]
-- For use in GitHub's native mobile apps [TODO: Screenshot of navbar and swipe actions (iOS)]
-- To show state [TODO: Screenshot of star and sponsor social buttons]
-
-### Status
-
-To show the status or completion of an action (e.g. an action running), the filled check and filled x icons can be used.
-
-[TODO: Show image of Actions]
+<Do>
+    <img width="456" alt="octicon-mobile-state" src="https://user-images.githubusercontent.com/10384315/103719160-4873ff80-4f7d-11eb-8253-634e1692d96b.png" />
+    <Caption>Use a filled icon to indicate navigation state in GitHub's native mobile apps</Caption>
+</Do>
```

Comment:
I think a "don't" example might be helpful here to show when not to use a filled icon.

Reply:
Looking at the set we have I can't think of a realistic example to show here. If you have an idea please feel free to share!